docs/cmf_client/step-by-step.md

```diff
@@ -7,7 +7,7 @@ Common metadata framework (cmf) has the following components:
 - **Central Artifact Repositories** host the code and data.
 
 ## Install cmf library i.e. cmflib
-Before proceeding, ensure that the CMF library is installed on your system. If not, follow the installation instructions provided inside the [CMF in a nutshell](../index.md) page.
+Before proceeding, ensure that the CMF library is installed on your system. If not, follow the installation instructions provided in the [Overview](../index.md) page.
```

Comment:
There is no page with 'Overview' heading. Please replace "Overview' with 'Home.'